mozetl/taar/taar_amodump.py

```diff
@@ -247,16 +247,16 @@ def marshal(value, name, type_def):
     serializers = {
         typing.List: list,
         typing.Dict: dict,
-        str: unicode,
-        unicode: unicode,
+        str: str,
+        str: str,
```

Comment:
Unicode is handled differently in python 3, so marshaling unicode isn't valid anymore. @crankycoder do you think this would affect the AMO dump, given that the tests pass in both version?